src/Tabs.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const Tabs = ({ tabs, selectedTab, onSelectedTab }) => (
+  <div>
+    <ul>
+      {tabs.map(tab => 
+      <li key={tab.id}>
+        <button type="button" onClick={() => {onSelectedTab(tab.id)}}>{tab.title}</button>
```

Comment:
Let's make your code more readable.

```suggestion
        <button
          type="button"
          onClick={() => {onSelectedTab(tab.id)}}
        >
          {tab.title}
        </button>
```